pawan/html/practice_assignment/contactform.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<HTML>
+	<HEAD>
+	    <TITLE>Contact Form</TITLE>
+    </HEAD>
+    <body>
+        <Form action="/home/pawanjagtap/MI-6-MEAN/pawan/html/practice_assignment/contactformdata.html" target="_blank">
```

Comment:
tell me, originally the spelling of form tag starts with the small letter, and you starts it with the capital letter. What will be the effect of this on your output?

Reply:
tell them the write way of writing the code in your comments

Reply:
nothing happens and it remains same